### DISCUSSION TITLE
Commit comments not disabling

### DISCUSSION THREAD

**Discussion by rawcomposition** [vercel/community#5054] [Help]
### Summary

I have Commit Comments toggled off, but I'm still getting comments. What's the deal?

See example commit: https://github.com/rawcomposition/birdinghotspots/commit/5010d8a43436cd659b8fdfb515a25fd5a3ad29ea

<img width="952" alt="Screenshot 2023-12-03 at 11 18 25 AM" src="https://github.com/vercel/community/assets/740090/81a67709-cb0b-48a8-a1a1-f9f8b7b7caa0">

### Example

https://github.com/rawcomposition/birdinghotspots/commit/5010d8a43436cd659b8fdfb515a25fd5a3ad29ea

### Steps to Reproduce

1. Disable commit comments
2. Make a new commit
3. See that a comment is still made by Vercel bot

**Comment by swarnava** [comment]
Hi,

Please can you confirm if this is part of a monorepo with multiple projects linked on Vercel?

**Comment by rawcomposition** [comment]
Not a monorepo, just a normal Github repo with one Next project

**Comment by ofhouse** [comment]
@rawcomposition Thanks for reporting!
We implemented a fix, can you confirm if this is still an issue for you?

**Reply by marcofugaro** [reply]
Still an issue for us, Vercel is commenting on commits even though that settings is disabled.
